DNN-OpenCV-Classification-with-Java/README.md

```diff
@@ -0,0 +1,190 @@
+This contains the code for **Image Classification with OpenCV Java**. For more information - visit [**Image Classification with OpenCV Java**](https://www.learnopencv.com/image-classification-with-opencv-java/)
+
+
+## Getting Started
+
+Our code is tested using Python 3.7.5, but it should also work with any other python3.x. If you'd like to check your
+version run:
+
+```bash
+python3 -V
+```
+
+### Virtual Environment
+
+Let's create a new virtual environment. You'll need to install [virtualenv](https://pypi.org/project/virtualenv/)
+package if you don't have it:
+
+```bash
+pip install virtualenv
+```
+
+Now we can create a new virtualenv variable and call it `env`:
+
+```bash
+python3 -m venv ~/env
+```
+
+The last thing we have to do is to activate it:
+
+```bash
+source  ~/env/bin/activate
+```
+
+To install the required python dependencies run:
+
+```bash
+pip3 install -r requirements.txt
+```
+
+### OpenCV
+
+In this blog post we are using OpenCV 4.3.0 unavailable via `pip` and OpenCV for Java. That is why, we first need to build the
+OpenCV library. To do so:
+
+1. Check the list of the below libraries. Install the missed dependencies:
+
+```bash
+sudo apt-get update
+sudo apt-get install build-essential cmake unzip pkg-config
+sudo apt-get install libjpeg-dev libpng-dev libtiff-dev
+sudo apt-get install libavcodec-dev libavformat-dev libswscale-dev libv4l-dev
+sudo apt-get install libxvidcore-dev libx264-dev
+sudo apt-get install libgtk-3-dev
+sudo apt-get install libatlas-base-dev gfortran
+sudo apt-get install python3-dev
+```
+
+For OpenCV Java installation we used default Java Runtime Environment and Java Development Kit:
+
+```bash
+sudo apt-get install default-jre
+sudo apt-get install default-jdk
+sudo apt-get install ant
+```
+
+2. Download the latest OpenCV version from the official repository:
+
+```bash
+cd ~
+wget -O opencv.zip https://github.com/opencv/opencv/archive/4.3.0.zip
+```
+
+3. Unzip the downloaded archives:
+
+```bash
+unzip opencv.zip
+```
+
+4. Rename the directories to match CMake paths:
+
+```bash
+mv opencv-4.3.0 opencv
+```
+
+5. Compile OpenCV. Create and enter a build directory:
+
+```bash
+cd ~/opencv
+mkdir build
+```
+
+6. Run CMake to configure the OpenCV build. Don't forget to set the right pass to the ``PYTHON_EXECUTABLE``:
```

Comment:
For my test, OpenCV was not able to find `ant`
Adding `-D ANT_EXECUTABLE=/usr/bin/ant` would be good.